### PR TITLE
chore: updated script to assign users with integration realm role

### PR DESCRIPTION
**Pull request by NithinKuruba**

### CODE REVIEW

File: kc-scripts/src/migrations/migrate-to-standard.ts [hunk 1]
```diff
@@ -90,6 +99,25 @@ container(async (baseAdminClient?: KeycloakAdminClient, targetAdminClient?: Keyc
...
 
+    console.log('re-match Gold standard users after migrating missing users');
+    userReport[i] = await matchTargetUsers(baseAdminClient, targetAdminClient, {
+      baseRealm,
+      targetRealm,
+      baseUsers: userList,
+    });
+
+    let validBaseTargetUsers = userReport[i]['valid-base-target-users'];
+    delete userReport[i]['valid-base-target-users'];
```
Comment (thegentlemanphysicist) [inline kc-scripts/src/migrations/migrate-to-standard.ts:110]: What does deleting this userReport do here?  Just trying to figure out why we delete if we're looping through the reports

Comment (NithinKuruba) [reply kc-scripts/src/migrations/migrate-to-standard.ts:110]: User report gives out stats about which users were successfully migrated and failed and the valid base target users are solely used for code purposes and we can delete it.

Comment (thegentlemanphysicist) [reply kc-scripts/src/migrations/migrate-to-standard.ts:110]: Ah that makes sense, thank you

**Review by thegentlemanphysicist — APPROVED** [review]
Looks good to me.

**Pull request merged**